Proxy/proxy.py

```diff
@@ -71,10 +72,20 @@ def uart_worker(modem, getDicts, units):
                         # Use new buffers
                         try:
                             getDicts[unit][port].append(item)
+
+                            # Check for Proxy logging and save to SQL if true
+                            if log:
+                                item["port"] = port
+                                sqlInsert(item)
                         except:
                             getDicts[unit][port] = deque([], maxlen=100)
                             getDicts[unit][port].append(item)
 
+                            # Check for Proxy logging and save to SQL if true
```

Comment:
If we log `item` in either case why not move this to after the try/except block.